src/test/java/com/rultor/agents/daemons/StartsDaemonITCase.java

```diff
@@ -118,40 +115,35 @@ public void startsDaemon() throws Exception {
     /**
      * StartsDaemon can deprecate default image.
      * @throws IOException In case of error
-     * @todo #1018:30min Implement a deprecation message at the start and end
-     *  of the process if the project is using the default image
-     *  'yegor256/rultor'. Fix the issue using the xpath available in
-     *  com.rultor.agents.daemons.StartsDaemon and
-     *  com.rultor.agents.daemons.EndsDaemon.
      * @todo #1018:30min This test should not check for start and end
      *  deprecation messages if when the repo is the actual Rultor repo:
      *  https://github.com/yegor256/rultor
      */
     @Test
-    @Ignore
     public void deprecatesDefaultImage() throws IOException {
-        final Talk talk = this.talk();
-        final XML xml = talk.read();
-        for (final String path
-            : xml.xpath("/p/entry[@key='merge']/entry[@key='script']")
-             ) {
-            if ("yegor256/rultor".equals(path)) {
-                final String dir = talk.read()
-                    .xpath("/talk/daemon/dir/text()").get(0);
-                MatcherAssert.assertThat(
-                    dir,
-                    StringStartsWith.startsWith("#### Deprecation Notice ####")
-                );
-            }
-        }
+        final Profile profile = this.profile(
+            new XMLDocument(
+                StringUtils.join(
+                    "<p>",
+                    "<entry key='docker'>",
+                    "<entry key='image'>yegor256/rultor</entry>",
+                    "</entry>",
+                    "</p>"
+                )
+            )
+        );
+        final Talk talk = this.talk(profile);
+        final String dir = talk.read().xpath("/talk/daemon/dir/text()").get(0);
+        MatcherAssert.assertThat(dir, Matchers.notNullValue());
```

Comment:
@mkordas This test case is involved in 2 different pieces of the same puzzle, if it was for this task only I would have removed it as I did for EndsDaemonITCase.deprecatesDefaultImage. So the only thing I did here, is to make it at least consistent for the next task because it was totally incorrect.